### ISSUE TITLE
Does this work?

### ISSUE THREAD

**Issue by bionichuman** [BenNottelling/Netflix-Proxy#1]
Does this work? Trying to find a web proxy for netflix.

**Comment by bionichuman** [comment]
Doesnt seem to work for me. I tried your code. Shows main page but login or anything else doesnt work.

**Comment by BenNottelling** [comment]
No this does not work, but I may update it soon so it works better, the base script I use for a proxy (miniproxy) was holding it back, but my fork of it can actually get you logged in now

**Comment by BenNottelling** [comment]
Ok. so, here's an update: 
By using this proxy script https://github.com/BenNottelling/miniProxy/tree/master you can actually login and browse netflix, but you can't watch anything because once you pick something to watch all you get is a black screen due to the proxy not supporting (and thus breaking) websockets. 
Now, it's very slow to due use that because ALL of the data has to be downloaded by your server, processed, and then sent to you. My proxy here is designed to make it faster by storing static items such as images on your server and by having mostly all of the page processing done. The rest of the work is then done by Miniproxy. It'll take me some time to update all of the pages in this repository, but I have a six hour car ride tommorrow so i'll likely have it done then
After that, I'll look into finding a workaround for the websocket issue.

Long story short though, no, it does not work, but It will get better. Feel free to ask questions or follow the repository here on github to keep track of the updates.

**Comment by bionichuman** [comment]
Wow, thanks. I really need this. I couldn’t figure out how the videos work and web socket makes sense.

Do you have an email or discord I could contact you? I would like to pay you to get this to work for me. Thanks.
